example/index.html

```diff
@@ -16,6 +20,8 @@
 <ol id="results">
 	<li>Add rectangles and then use search to populate the results list</li>
 </ol>
+<button id="graphviz">Generate GraphViz</button> <button id="copy">Copy Results</button>
```

Comment:
I'm not sure this is worth keeping in. I had some thoughts about trying to visualize how "balanced" the rtree is, how much overlap, etc. This will help visualize the balance in terms of child nodes per node, but disregards the bounding boxes of the nodes which is more important in most cases.